src/main/java/org/sentrysoftware/tablejoin/TableJoinClient.java

```diff
@@ -0,0 +1,394 @@
+/*-
+ * ╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲╱╲
+ * MY PROJECT
```

Comment:
I guess you've generated the header before modifying the pom.xml. Consequently, the header does not contain the real project name "TableJoin Client. You can re-run the header generation command to update it.